kpconv_torch/utils/metrics.py

```diff
@@ -23,9 +23,9 @@ def fast_confusion(true, pred, label_values=None):
                 len(pred.shape)
             )
         )
-    if true.dtype not in [np.int32, np.int64]:
+    if true.dtype not in [np.int8, np.int16, np.int32, np.int64]:
         raise ValueError(f"Truth values are {true.dtype} instead of int32 or int64")
```

Comment:
"instead of int8, int16, int32 or int64"

Reply:
Fixed.